### PR TITLE
Typo in Makefile template

### PR DESCRIPTION
**Pull request by kalbermattenm**
allways -> always

I just hope that I understood the whole context of this sentence...

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -444,7 +444,7 @@ compile-py-catalog: $(SERVER_LOCALISATION_FILES)
 
 $(PACKAGE)/locale/$(PACKAGE)-server.pot: $(SERVER_LOCALISATION_SOURCES_FILES) $(VENV_BIN)/pot-create lingua.cfg
 	$(VENV_BIN)/pot-create -c lingua.cfg -o $@ $(SERVER_LOCALISATION_SOURCES_FILES)
-	# removes the allways changed date line
+	# always remove date lines which have changed
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: always is about changed not remove ...
=> ?
remove date lines which are always changing

Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: +1 for `Remove date lines which are always changing.`

**Comment by kalbermattenm** [comment]
I just update my commit.

Thanks a lot @sbrunner and @ger-benjamin

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**